### DISCUSSION TITLE
Adding number row to keymap

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1777] [Ideas]
It would be nice to be able to see the number row on the keymap when typing numbers.

**Comment by frogfrogfrogfrogfrog** [comment]
I would like something like this but rather than the numbers I would love for their shift counterparts (! @ # $ % ^ & *) to be there as well since I figure that's a better use of the number row since you can already estimate where the keys are.

Also having a fourth row make the keymap sorta clunky so perhaps you have to enable it?

**Reply by BentEngbers** [reply]
maybe it can look something more like keybr? (https://www.keybr.com)

**Comment by TheSeriousProgrammer** [comment]
The keyboard layout currently has only letters in it , so it becomes difficult when I want to start practicing my special characters. Hope this will be implemented soon. Big Fan Here !!
